functorch/_src/eager_transforms.py

```diff
@@ -150,6 +166,14 @@ def _slice_argnums(args, argnums):
         return tuple(_safe_index(args, i) for i in argnums)
     raise RuntimeError(f'argnums must be int or Tuple[int, ...], got: {type(argnums)}')
 
+def _build_wrapper(f, args, argnums):
```

Comment:
Might be a good idea to rename this something more informative - I could see this being a more generically useful function.